tests/interactive/cmd_add_binding_test.go

```diff
@@ -0,0 +1,104 @@
+//go:build linux || darwin || dragonfly || solaris || openbsd || netbsd || freebsd
+// +build linux darwin dragonfly solaris openbsd netbsd freebsd
+
+package interactive
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/redhat-developer/odo/tests/helper"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("odo init interactive command tests", func() {
+
+	var commonVar helper.CommonVar
+	var serviceName string
+
+	// This is run before every Spec (It)
+	var _ = BeforeEach(func() {
+		if helper.IsKubernetesCluster() {
+			Skip("Operators have not been setup on Kubernetes cluster yet. Remove this once the issue has been fixed.")
```

Comment:
Really? Why is that the case? I mean, what's blocking us from setting up Operators on k8s cluster?